### PR TITLE
Add CIFAR-10 and CIFAR-100 example and Dropout Layer

### PR DESCRIPTION
**Pull request by workofart**
## Description

- Add CIFAR-10
- Add CIFAR-100 (using the same model architecture as CIFAR-10 just 100 classes as output)
- Add Dropout layer as a low hanging fruit to improve model performance

## Tests
Added unit tests for Dropout Layer.

I know the accuracy is a little low, this is mostly because we are not using convolutional layers but just fully connected  layers + relu activations. I will add convolutional layers in another PR and compare the performance. This PR is just to demonstrate the capability of building a neural network that tackle the CIFAR-10 and CIFAR-100 problems. The [best one achieves 70% test accuracy](https://arxiv.org/abs/1511.02580) on CIFAR-10 without convolutional layers.

**CIFAR-10**
image

**CIFAR-100**
image

### CODE REVIEW

File: autograd/utils.py [hunk 1]
```diff
@@ -6,8 +6,6 @@
 
 
 def train_test_split(X, y, test_size=0.2, random_state=None):
-    np.random.seed(random_state or 1337)
```
Comment (workofart) [inline autograd/utils.py:9]: We will remove this since this util will be used for training actual models with dropout layers
